js/apps/admin-ui/src/realm-settings/user-profile/AttributesTab.tsx

```diff
@@ -120,7 +126,8 @@ export const AttributesTab = ({ setTableData }: AttributesTabProps) => {
         );
 
         const updatedAttributes = config.attributes.filter(
-          (attribute) => attribute.name !== attributeToDelete,
+          (attribute: UserProfileAttribute) =>
+            attribute.name !== attributeToDelete,
```

Comment:
Same here, type can be inferred.
```suggestion
          (attribute) => attribute.name !== attributeToDelete,
```